infrastructure/cdn-in-a-box/cache/run.sh

```diff
@@ -17,14 +17,78 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set -e
-set -x
-set -m
+set -o errexit -o xtrace -o monitor
+
+mkdir /tmp/ort
+
+set-dns.sh
+insert-self-into-dns.sh
+
+source /to-access.sh
+
+# Wait on SSL certificate generation
+until [[ -f "$X509_CA_ENV_FILE" ]]
+do
+  echo "Waiting on Shared SSL certificate generation"
+  sleep 3
```

Comment:
Inconsistent indentation using 2 spaces - existing code in this file used 4 spaces. I wouldn't object to switching to tabs (tabs > 4-space > 2-space >>> 8-space), but it should be done consistently and globally through the file